live/global.ini

```diff
@@ -22891,113 +22891,113 @@ PU_BANISTER_MG_Celebrate_IG_003_LoveIt=Ich liebe es!
+PU_BANISTER_MG_Intro_1stX_IG_001_HiWelcomeTo=Hallo! Willkommen in der Nachbarschaft. Tessa Banister, Junior-Kartographin 2. Klasse, meldet sich. Ich bin das Auge hinter all den ICC-Scan-Sonden in diesem Gebiet. Was diese kleinen Kerle sehen, sehe ich auch. Was, ja, bedeutet, dass ich Dich absolut sehen kann. Also... Nun... <Pause> Wie auch immer, es gab einen Grund, warum ich mich gemeldet habe. Ich darf meinen Posten nicht verlassen, weil ich einen wichtigen Sprungpunkt scannen muss und so, also habe ich gehofft, Du könntest mir bei etwas helfen.
+PU_BANISTER_MG_LastMission_Beacon_FailStreak_IG_001_AnotherFailedRescue=Wieder eine gescheiterte Rettungsaktion, was? Sie sind doch nicht etwa einem Schamanen begegnet, oder? Nicht, dass ich an Flüche glaube, aber angesichts der überwältigenden Beweise könnte es nicht schaden, diese Haltung zu überdenken.
+PU_BANISTER_MG_LastMission_Beacon_FailStreak_IG_002_WeKeepTrying=Wir versuchen immer wieder, den Menschen in den Notunterkünften zu helfen, aber was nützt das, wenn diese Verbrecher immer skrupelloser werden. Vielleicht müssen wir akzeptieren, dass es ein hoffnungsloser Fall ist.
+PU_BANISTER_MG_LastMission_Beacon_FailStreak_IG_003_HowManyIs=Wie viele haben wir jetzt schon verloren? Nach dem letzten Notsignal frage ich mich langsam, warum wir es überhaupt versuchen.
+PU_BANISTER_MG_LastMission_Beacon_FailTurnAround_IG_001_ReallyAShame=Das mit dem letzten Notsignal ist wirklich schade, aber man kann ja nicht immer gewinnen. Trotzdem traurig.
+PU_BANISTER_MG_LastMission_Beacon_FailTurnAround_IG_002_ListenIKnow=Ich weiß, dass wir den letzten Piloten verloren haben, aber darauf darf man sich nicht konzentrieren. Du hast vielen Menschen da draußen geholfen.
+PU_BANISTER_MG_LastMission_Beacon_FailTurnAround_IG_003_OnThatLast=Gib Dir bei dem letzten Notsignal nicht die Schuld an dem, was passiert ist. Du wolltest nur helfen.
+PU_BANISTER_MG_LastMission_Beacon_Fail_IG_001_EvenThoughYou=Auch wenn Du das Schiff nicht retten konntest, war es sehr nett, dass Du es versucht hast.
+PU_BANISTER_MG_LastMission_Beacon_Fail_IG_002_IHeardWhat=Ich habe gehört, was mit dem letzten Notruf passiert ist. Das war ein Schlamassel.
+PU_BANISTER_MG_LastMission_Beacon_Fail_IG_003_ItWasToo=Das mit dem Schiff, dem Du helfen wolltest, ist wirklich schade. Ich schätze, manchmal ist unser Bestes einfach nicht gut genug.
+PU_BANISTER_MG_LastMission_Beacon_Fail_IG_004_IWasSorry=Das mit dem letzten Schiff hat mir leid getan. Das ist echt scheiße. Wenigstens geht es Dir gut.
+PU_BANISTER_MG_LastMission_Beacon_Fail_IG_005_WantedToLet=Ich wollte Dich wissen lassen, dass ich es zu schätzen weiß, dass Du Dein Leben riskiert hast, um diesen Menschen zu helfen, auch wenn es so endete, wie es endete.
+PU_BANISTER_MG_LastMission_Beacon_Ignored_IG_001_IWasA=Ich war etwas überrascht, dass Du dem letzten Piloten nicht helfen wolltest. Es war wohl dumm von mir, dass ich dachte, Du seist jemand, der sich um Menschen in Schwierigkeiten kümmert.
```

Comment:
Vorschlag: `Ich war etwas überrascht, dass Du dem letzten Piloten nicht helfen wolltest. Es war wohl dumm von mir zu glauben, du seist jemand, der sich um Menschen in Schwierigkeiten kümmert.`